ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -1032,21 +1041,22 @@ protected static void endToEndReplicationWithMultiNodeSinglePartitionTest(String
 
       // get blob
       ids.clear();
-      ids.add(blobId1);
+      ids.add(blobIdList.get(0));
       GetRequest getRequest3 =
           new GetRequest(1, "clientid2", MessageFormatFlags.Blob, partitionRequestInfoList, GetOption.None);
       channel3.send(getRequest3);
       stream = channel3.receive().getInputStream();
       GetResponse resp3 = GetResponse.readFrom(new DataInputStream(stream), clusterMap);
       //System.out.println("response from get " + resp3.getError());
       try {
+        // todo: fix once router support encryption. Might have to decrypt to compare data
```

Comment:
if we are putting without router involvement (like we are in this test), why would this happen?